File: stacks/nodejs-express/backend/src/routes/hello.ts
import { Router, Request, Response } from "express";

const router = Router();

router.get("/api/hello", (_req: Request, res: Response) => {
  res.json({
    message: "Hello from Express!",
    lang: "node",
    version: process.version.replace("v", ""),
  });
});

export default router;

File: stacks/python-fastapi/backend/src/routers/hello.py
import sys

from fastapi import APIRouter

router = APIRouter()


@router.get("/api/hello")
async def hello():
    version = f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}"
    return {
        "message": "Hello from FastAPI!",
        "lang": "python",
        "version": version,
    }
